### ISSUE TITLE
Content on home page is not a real header but styled as one

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4036]
## Area or Component
SimpleReport home page

## Quick Summary
The tagline "A better way to report COVID-19 rapid tests" is stylized as a header, but doesn't describe any content that follows.

## Accessibility Violation
non-heading text should not be marked as a heading for decorative purposes: https://dequeuniversity.com/class/semantic-structure/headings/real-headings#headings-for-semantic-purposes-only

## Action Requested
Change this text to be a big `<span>` element or similar, instead of a heading

## Screenshots (optional)
![Screen Shot 2022-07-21 at 5 54 04 PM](https://user-images.githubusercontent.com/80282552/180338880-f9c468f7-b813-4a19-8636-c88c02490025.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=856291185&range=36:36
